common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,192 @@ def test_teams_membership(self):
             self.browse_teams_page.team_cards[0].find_element_by_css_selector('.member-count').text,
             '1 / 10 Members'
         )
+
+    def test_navigation_links_are_working_properly(self):
+        """
+        Scenario: User should be able to navigate to "browse all teams" and "search team description" links.
+        Given I am enrolled in a course with a team configuration and a topic
+            containing one team
+        When I visit the Teams page for that topic
+        Then I should see the correct page header
+        And I should see the link to "browse all team"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        And I should see the link to "search teams"
+        And I should navigate to that link
+        And I see the relevant page loaded
+        """
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+
+        self.assertTrue(self.browse_teams_page.browse_all_teams_link_present())
+        self.browse_teams_page.click_browse_all_teams_link()
+        self.topics_page.is_browser_on_page()
+
+        self.browse_teams_page.visit()
+        self.verify_page_header()
+        self.assertTrue(self.browse_teams_page.search_team_link_present())
+        self.browse_teams_page.click_search_team_link()
+        # TODO Add search page expectation once that implemented.
+
+
+@attr('shard_5')
+class CreateTeamTest(TeamsTabBase):
+    """
+    Tests for creating a new Team within a Topic on the Teams page.
+    """
+
+    def setUp(self):
+        super(CreateTeamTest, self).setUp()
+        self.topic = {u"name": u"Example Topic", u"id": "example_topic", u"description": "Description"}
+        self.set_team_configuration({'course_id': self.course_id, 'max_team_size': 10, 'topics': [self.topic]})
+        self.browse_teams_page = BrowseTeamsPage(self.browser, self.course_id, self.topic)
+        self.browse_teams_page.visit()
+        self.create_team_page = CreateTeamPage(self.browser, self.course_id, self.topic)
+
+    def verify_page_header(self):
+        """
+        Verify that the page header correctly reflects the
+        create team header, description and breadcrumb.
+        """
+        self.assertEqual(self.create_team_page.header_page_name, 'New Team')
+        self.assertEqual(
+            self.create_team_page.header_page_description,
+            'Create a new team if you can\'t find existing teams to join, '
+            'or if you would like to learn with friends you know.'
+        )
+        self.assertEqual(self.create_team_page.header_page_breadcrumbs, 'Example Topic')
+
+    def verify_and_navigate_to_create_team_page(self):
+        """Navigates to the create team page and verifies."""
+        self.create_team_page.wait_for_ajax()
```

Comment:
I don't understand the need for this line. We're not on the Create Team page at the time this method is called, so why should we call its `wait_for_ajax()` method?